draft-ietf-quic-transport.md

```diff
@@ -470,7 +472,9 @@ this version, it contains:
   * 01: 1-RTT packet (packet number size = 1)
   * 02: 1-RTT packet (packet number size = 2)
   * 03: 1-RTT packet (packet number size = 4)
+  * 07: 1-RTT packet (packet number size = 4, with packet number echo)
```

Comment:
Sadly, I think a 1 byte echo is commonly too small, whereas 2 bytes is almost always sufficient.  Given this is going to be used on the public internet, I'd define the 2 byte version but not the 1 byte version.